mosaic_api/mosaic_api/api/api_v1/endpoints/mosaics.py

```diff
@@ -0,0 +1,188 @@
+"""Mosaic endpoints."""
+from mosaic_api.models.mosaics import Mosaic, MosaicRequest, Link
+import random
+import string
+from fastapi import APIRouter, Response, HTTPException
+from starlette import status
+from starlette.requests import Request
+from cogeo_mosaic.mosaic import MosaicJSON
+from rio_tiler.io import COGReader
+from pystac_client import Client
+from typing import List, Optional
+from asyncio import wait_for, create_task
+import asyncio
+import aiohttp
+
+from mosaic_api.core.config import MOSAIC_API_ROOT
```

Comment:
I would suggest to use iSort to sort the module imports, but this is not mandatory